projects/docs/docs/commands/plugin.md

```diff
@@ -0,0 +1,46 @@
+---
+slug: '/commands/plugin'
+title: Interact with plugins
+description: 'Learn how to use plugin commands for developing and publishing tuist plugins.'
+---
```

Comment:
This is just for Tasks, right? I would specify it doesn't apply to project description helpers